lib/razor/config.rb

```diff
@@ -43,7 +43,7 @@ def initialize(env, fname = nil, defaults_file = nil)
       # Use the filename given, or from the environment, or from /etc if it
       # exists, otherwise the one in our root directory...
       fname ||= ENV["RAZOR_CONFIG"] ||
-        (File.file?('/etc/razor/config.yaml') and '/etc/razor/config.yaml') ||
+        (File.file?('/etc/puppetlabs/razor-server/config.yaml') and '/etc/puppetlabs/razor-server/config.yaml') ||
```

Comment:
As it stands, no.

There are pathing changes required to the config file for it to work (see above). So, what the packaging does it take the updated config.yaml and drops it into place. I was hoping we would update the docs to suggest a partial migration where the user effectively ensures any updates to their config file are moved over to the new one. We could probably move the old file and just sed out any of the pathing updates we know need to happen. I'm not sure when I'll have time to take care of that though